plugins/hypervisors/xenserver/src/com/cloud/hypervisor/xenserver/resource/Xenserver625StorageProcessor.java

```diff
@@ -100,36 +100,25 @@ protected SR createFileSR(final Connection conn, final String path) {
         PBD pbd = null;
 
         try {
-            final String srname = hypervisorResource.getHost().getUuid() + path.trim();
-
-            final Set<SR> srs = SR.getByNameLabel(conn, srname);
-
-            if (srs != null && !srs.isEmpty()) {
-                return srs.iterator().next();
+            final String srname = path.trim();
```

Comment:
@abhinandanprateek could there be a case that after upgrade SR.getByNameLabel will fail as the srname has changed? I would say let's not break the naming convention if it's used in/by external resources.